docs/code/squash_and_merge.py

```diff
@@ -32,25 +33,45 @@
 dataset_client.commit("commit-4")
 """"""
 
-"""Squash and Merge"""
-draft_number = dataset_client.squash_and_merge(
-    "draft-5",
-    description="description",
+"""Create Job"""
+squash_and_merge = dataset_client.squash_and_merges
+job = squash_and_merge.create_job(
+    draft_title="draft-5",
     source_branch_name="dev",
     target_branch_name="main",
+    draft_description="draft_description",
     strategy="override",
 )
-dataset_client.checkout(draft_number=draft_number)
-dataset_client.commit("commit-5")
 """"""
 
 """Checkout First"""
-draft_number = dataset_client.squash_and_merge(
-    "draft-5",
-    description="description",
+job = squash_and_merge.create_job(
+    draft_title="draft-5",
     source_branch_name="dev",
+    draft_description="draft_description",
     strategy="override",
 )
-dataset_client.checkout(draft_number=draft_number)
-dataset_client.commit("commit-5")
+""""""
+
+"""Get, List and Delete"""
+job = squash_and_merge.get_job("jobId")
```

Comment:
shall we add `<>` around `"jobId"`

Reply:
but we add `<>` for `accesskey`
```python
gas = GAS("<YOUR_ACCESSKEY>")
```
I think it is a problem, we need to fix it globally.

Reply:
Ok, the task will be set up later to modify it globally.